openshift_cli_installer/libs/clusters/rosa_cluster.py

```diff
@@ -138,7 +138,6 @@ def destroy_hypershift_vpc(self) -> None:
         )
         if rc != 0:
             self.logger.error(f"{self.log_prefix}: Failed to destroy hypershift VPCs with error: {err}")
-            raise click.Abort()
```

Comment:
Why did you remove this raise? don't we want/need to exit here?

Reply:
so the destroy flow will continue and not exit (e.g oidc deletion)